index.html

```diff
@@ -7,41 +7,64 @@
     <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <nav>
-        <div class="name">Iwona</div>
-        <div class="list">
-            <span>01. à propos</span> 
-            <span>02. experience</span> 
-            <span>03. projects </span>
-            <span>04. contact</span>
-            
-        </div>
+
+      <nav>
```

Comment:
troszkę nie rozumiem tutaj budowy klas, bo ten div na dole class="preheader" tak powinien być? 

zrobiłabym to tak 

```
<nav class="nav">
    <div class="nav__container">
        <a class="nav__container___link">01.</a>
        <a class="nav__container___link">02.</a>
    </div>
</nav>

```

Pytanie tylko czy ten div nav__container będzie Ci potrzebny :) Bo jeśli nie to staramy się nie używać nadmiarowo divów

Reply:
jeśli byłby nie potrzebny to zrobiłabym 
```
<nav class="nav">
        <a class="nav__link">01.</a>
        <a class="nav__link">02.</a>
</nav>
```